addons/horizontal-mystuff-tabs/addon.json

```diff
@@ -2,6 +2,7 @@
   "name": "Horizontal My Stuff tabs",
   "description": "Moves the My Stuff navigation buttons (\"All Projects\", \"My Studios\", \"Trash\") to the top of the page, instead of keeping them on the left.",
   "tags": ["community", "theme"],
+  "relatedAddons": ["scratchr2"],
```

Comment:
This addon used to be a part of 2.0 → 3.0, but it shouldn't have been. The relation implies that making the tabs horizontal makes the My Stuff page more 3.0-like, even though we can't possibly know if that's true because there's no 3.0 version of the My Stuff page.

Reply:
> wasn't the whole purpose of the addon to revert back to the old behavior of 2.0 -> 3.0 on the my stuff page?

It was, but I made it a separate addon instead of adding the feature back to 2.0 → 3.0 specifically because it isn't really related to 2.0 → 3.0. I don't think it would make sense to make all addons that used to be a part of devtools related to each other either.